pkgs/by-name/wa/warp-terminal/package.nix

```diff
@@ -64,6 +67,10 @@ linux = stdenv.mkDerivation (finalAttrs:  {
 
     runHook postInstall
   '';
+
+  postInstall = ''
+    wrapProgram $out/bin/warp-terminal --set WARP_ENABLE_WAYLAND 1
```

Comment:
what happens if this flag is set and wayland is not available? i'd suggest not wrapping so the behavior of the program is identical to what is in the docs: https://docs.warp.dev/help/known-issues#linux-1 or adding some booleans or something. idk.

Reply:
From what I can tell, it fallsback to x11, this just sets wayland priority higher. If I run it on Xorg it's being picked up by xeyes as a proper x11 app. The thing that's linked to the known-issues you sent seems to be for WSL on windows, but I also saw this? https://github.com/warpdotdev/Warp/issues/4628. I'm confused on whether or not wayland is stable. If it isn't then I agree that it should just be a flag and then we can set it to default later.

Reply:
https://github.com/warpdotdev/Warp/issues/4240#issuecomment-1961993281 although i have no issue warp considers it unstable so its  not a bad idea to make it opt in rather than the default. It wayland support is very usable as of the last few updates

Reply:
Yeah, I saw that, but I was unsure about the wording. Is it unstable in WSL or just wayland in general?